common/persistence/sql/testutil.go

```diff
@@ -77,6 +77,7 @@ func dropDatabase(db *sqlx.DB, dbName string) (err error) {
 func loadDatabaseSchema(dir string, fileNames []string, db *sqlx.DB, override bool) (err error) {
 
 	for _, file := range fileNames {
+		// #nosec
```

Comment:
@longquanzheng please review and provide comment.

Reply:
It only load schema file for local testing.